src/app/vendors/d2-vendors.ts

```diff
@@ -40,7 +40,7 @@ const vendorOrder = [
   VENDORS.BENEDICT,
   VENDORS.BANSHEE,
   VENDORS.DRIFTER,
-  VENDORS.ADA,
+  VENDORS.ADA_FORGE,
```

Comment:
cc @bhollis -- two of these vendors are deprecated, and Spider is in a different category now. What was the rationale behind the order back then?

Reply:
I think in the beginning it was just because the order from the API didn't make much sense compared to in-game. But we've done a lot about sorting and categorization since then, so this may no longer be relevant.